zoninator.php

```diff
@@ -171,6 +175,27 @@ function admin_page_init() {
 		add_menu_page( __( 'Zoninator', 'zoninator' ), __( 'Zones', 'zoninator' ), $this->_get_manage_zones_cap(), $this->key, array( $this, 'admin_page' ), '', 11 );
 	}
 
+	/**
+	 * Loads any admin notices on the top level zones screen.
+	 */
+	public function admin_notices() {
+
+		$screen = get_current_screen();
+
+		if ( 'toplevel_page_zoninator' === $screen->base ) {
+
+			if ( isset( $_GET['zone_lock'] ) ) {
+				$zone = $this->get_zone( intval( $_GET['zone_lock'] ) );
+			?>
+				<div class="notice notice-warning is-dismissible">
+					<p><?php echo sprintf( $this->_get_message('error-zone-lock-redirect'), esc_html( $zone->name ) ); ?></p>
```

Comment:
What happens here if they pass a zone_id that doesn't exist?

Reply:
@sboisvert Ah, yeah, I'll need to only display the notice if `$zone` is not `false`.

Do you think there should be a different notice if `$_GET['zone_lock']` isn't a valid Zone ID or ignore it? I think that _should_ only be able to happen if:
1. The zone is deleted as the user who was editing the zone is being redirected
2. Someone purposefully puts an invalid Zone ID in the `zone_lock` url param

Reply:
Nod I agree on when it should only happen. I suspect that this will throw a PHP Warning if that's the case since we're using the return of $this->get_zone() without checking if it's actually what we expect as per https://vip.wordpress.com/documentation/code-review-what-we-look-for/#not-checking-return-values :)

Reply:
Correct. It does throw a warning there if you pass an invalid Zone ID into `$_GET['zone_lock']`.

Thoughts on adding a _different_ notice if the Zone ID is invalid?